mm/kfence/kfence-test.c

```diff
@@ -625,15 +625,51 @@ static void test_krealloc(struct kunit *test)
 	KUNIT_ASSERT_TRUE(test, report_matches(&expect));
 }
 
+/* Test that some objects from a bulk allocation belong to KFENCE pool. */
+static void test_alloc_bulk(struct kunit *test)
+{
+	const size_t size = 32;
+	void *objects[100];
+	bool pass = false;
+	int i, num;
```

Comment:
Move 'i,num' into the do-loop, and just initialize num straight away?

Reply:
Done